client/layout/guided-tours/tours/gdocs-integration-tour.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	LinkQuit,
+	Quit,
+} from 'layout/guided-tours/config-elements';
+import {
+	hasUserPastedFromGoogleDocs,
+	isEnabled
+} from 'state/ui/guided-tours/contexts';
+
+export const GDocsIntegrationTour = makeTour(
+	<Tour
+		name="gdocsIntegrationTour"
+		version="20170124"
+		path="/post"
+		when={ and(
+			isEnabled( 'guided-tours/gdocs-integration-tour' ),
+			hasUserPastedFromGoogleDocs
+		) }
+	>
+		<Step name="init" placement="right">
+			<p> { translate( 'Are you interested in posting drafts directly from Google Docs?' ) }</p>
+			<ButtonRow>
+				<LinkQuit
+					primary
+					target="_blank"
+					href="https://wordpress.com">
+					{ translate( 'Learn more' ) }
+				</LinkQuit>
+				<Quit>{ translate( 'No, thanks' ) }</Quit>
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'No thanks' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/show-tour-gdocs)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).